### ISSUE TITLE
Version 0.0.4 not on crates.io

### ISSUE THREAD

**Issue by alexmullins** [zyantific/zydis-rs#12]
Hi,

I am trying to build a project with Zydis-rs. I am following the README and setting
```toml
[dependencies]
zydis = "0.0.4"
```
When I run ```cargo build``` I get the following error:
```bash
$ cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: no matching version `^0.0.4` found for package `zydis`
location searched: registry `https://github.com/rust-lang/crates.io-index`
versions found: 0.0.3, 0.0.1
required by package `nopstats v0.1.0 ({file path})`
```

Could the new version be pushed to crates.io?

Thanks

Alex

**Comment by alexmullins** [comment]
The work around I used for now is:

```toml
[dependencies]
zydis = { git = "https://github.com/zyantific/zydis-rs", tag = "v0.0.4" }
```

**Comment by th0rex** [comment]
Hello.

The state of version 0.0.4 is kind of strange right now. Initially we wanted to make a release, because there have been some refactorings and bug fixes, so everything was prepared for that release. But in the end I decided to not do a release since work is currently being done on zydis version 3, which would lead to another breaking release. Additionally, there have been some major changes to the bindings as well on the future branch. So once zydis version 3 is done, the current future branch will be released as version 0.0.4 and there likely won't be any major API changes after that (at least for as long as these bindings target zydis version 3).

I just updated the README on the master branch to use version 0.0.3 instead and mention the `future` branch as an option.

So I would suggest you either use the version 0.0.3 from crates.io, or you use the latest commit from the `future` branch, since what is tagged as v0.0.4 here is kind of an in between state of both, neither stable nor has it the newest API. I hope this didn't cause too much confusion.
